pkg/auto/chart_bump.go

```diff
@@ -245,6 +245,18 @@ func (b *Bump) BumpChart(ctx context.Context, versionOverride string) error {
 		return err
 	}
 
+	// check if the version to bump does not already exists
+	alreadyExist, err := checkBumpAppVersion(ctx, b.Pkg.UpstreamChartVersion, b.assetsVersionsMap[b.targetChart])
+	if err != nil {
+		logger.Log(ctx, slog.LevelError, "error while checking if version exists", logger.Err(err))
+		return err
+	}
+	if alreadyExist {
+		logger.Log(ctx, slog.LevelError, "version to bump already exists", slog.String("version", *b.Pkg.UpstreamChartVersion))
+		git.FullReset() // quitting the job regardless if this works or not
+		return fmt.Errorf("version to bump already exists: %s", *b.Pkg.UpstreamChartVersion)
```

Comment:
Generally, you want to avoid any format functions when using strings. They can be concatenation with the "+" operator and avoid all of the unnecessary reflection and memory use.